pages/common/deno.md

```diff
@@ -0,0 +1,25 @@
+# deno
+
+> A secure runtime for JavaScript and TypeScript
+> More information: <https://deno.land/>.
+
+- Run a JavaScript or TypeScript file:
+
+`deno file.ts`
+
+- Start a REPL (interactive shell):
+
+`deno`
+
+- Run a file with access to the network:
+
+`deno --allow-net server.ts`
+
+- Run a file from a URL:
+
+`deno https://deno.land/std/examples/welcome.ts`
```

Comment:
```suggestion
`deno {{file_url}}`
```

I'm not sure on this one, maybe it should be some other name for the parameter.

Reply:
What about this?
```suggestion
`deno https://example.com/std/examples/welcome.ts`
```

Reply:
We usually use full URLs where possible. Changing the URL to `example.com` doesn't make much sense since it becomes invalid. The provided URL is already ok, it just needs to be enclosed in curly braces `{{ }}`.